src/hip/CUDACore/radixSort.h

```diff
@@ -260,7 +262,9 @@ namespace cms {
   namespace hip {
 
     template <typename T, int NS = sizeof(T)>
-    __global__ void __launch_bounds__(256, 4)
+    // The launch bounds seems to cause the kernel to silently fail to run (rocm 4.3)
+    //__global__ void __launch_bounds__(256, 4)
```

Comment:
As I'm not familiar with HIP, I'm wondering: is the problem that these specific launch bounds do not work on the AMD GPU you tested -- or that launch bounds are not supported by HIP ?

Reply:
OK, sorry, found it in the HIP documenation: [launch bounds](https://rocmdocs.amd.com/en/latest/Programming_Guides/Kernel_language.html#launch-bounds).

Reply:
What are the launch parameters for `radixSortMultiWrapper` when it fails ?